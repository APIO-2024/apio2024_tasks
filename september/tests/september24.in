txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 1 2 3 4 6 6 7
9 5 8 7 3 4 6 2 1
10 1
0 0 1 3 4 5 6 6 7
2 8 6 9 7 5 3 4 1
10 1
0 0 1 2 4 4 6 7 7
6 9 1 4 8 5 3 2 7
10 1
0 0 2 2 4 5 5 7 8
7 4 3 5 2 9 1 8 6
10 1
0 0 2 2 3 4 6 7 7
1 6 2 7 5 3 4 9 8
10 1
0 1 2 2 4 4 6 6 8
7 3 5 9 8 6 4 1 2
10 1
0 0 2 2 4 5 6 6 8
7 3 1 9 8 4 5 2 6
10 1
0 0 1 2 4 5 6 7 7
8 3 1 9 7 6 4 5 2
10 1
0 0 1 2 4 5 6 7 8
3 9 1 5 8 4 7 6 2
10 1
0 0 1 3 4 5 6 6 8
7 2 9 6 8 5 3 1 4
10 1
0 0 2 3 3 5 6 7 8
1 4 9 8 6 7 5 2 3
10 1
0 0 1 2 3 5 6 7 7
8 7 9 6 5 2 3 4 1
10 1
0 1 1 2 4 4 5 7 7
3 9 6 8 7 5 4 2 1
10 1
0 0 2 2 3 4 6 7 8
1 5 3 4 6 2 8 9 7
10 1
0 0 1 3 4 4 6 6 8
9 5 2 8 4 6 7 3 1
10 1
0 0 2 2 3 4 5 6 7
1 9 7 8 5 3 6 4 2
10 1
0 1 1 2 4 4 5 6 7
7 1 3 2 5 4 6 8 9
10 1
0 1 2 2 4 5 6 6 7
9 3 8 6 7 5 4 2 1
10 1
0 1 1 2 3 4 5 7 8
9 6 7 8 4 2 5 3 1
10 1
0 0 1 2 3 5 6 6 7
9 8 4 2 6 7 3 5 1
40 1
0 1 1 3 2 3 5 7 8 8 9 9 10 11 14 13 16 15 16 19 18 19 22 21 24 23 26 25 27 29 29 31 31 33 32 33 36 37 36
6 28 39 38 37 25 34 20 35 12 4 17 36 33 32 3 31 24 21 30 29 18 15 14 11 9 27 23 26 19 22 16 10 13 8 7 5 2 1
40 1
0 0 0 2 2 4 4 6 7 7 9 11 11 13 13 14 15 16 16 17 20 21 22 21 23 23 25 26 27 28 28 30 31 31 33 33 34 36 36
38 29 1 19 18 8 37 10 3 12 24 6 39 16 14 34 36 35 27 32 5 25 30 33 31 28 26 23 22 21 20 17 13 15 11 9 4 7 2
40 1
0 0 0 2 3 4 4 6 8 9 9 10 10 12 13 13 16 16 17 17 19 20 22 23 22 25 26 25 26 27 28 30 31 32 33 34 36 36 36
1 15 11 21 7 5 24 35 3 18 33 37 31 23 28 14 39 19 38 12 34 36 32 30 27 29 26 25 22 20 17 16 13 10 9 6 4 8 2
40 1
0 1 2 3 2 4 5 5 8 8 9 11 11 13 12 15 15 16 16 19 19 19 21 23 23 24 26 27 27 28 29 30 32 31 34 35 35 37 37
22 39 10 7 18 14 33 13 36 20 17 6 3 4 25 38 35 37 34 32 31 30 29 27 28 26 24 23 21 19 15 16 12 11 8 9 5 2 1
40 1
0 0 0 2 4 5 5 7 7 8 10 11 11 13 14 15 15 16 18 19 18 21 20 23 22 24 25 27 26 29 28 31 32 33 32 34 34 37 36
30 29 39 12 17 3 9 1 26 24 36 35 23 6 20 38 19 37 34 32 33 31 28 25 22 27 21 16 18 15 14 11 13 10 5 7 8 4 2
100 1
0 1 2 3 1 4 4 6 6 7 9 11 10 12 14 12 13 14 18 16 17 20 20 21 21 25 24 24 25 28 30 31 29 33 34 34 36 37 36 36 39 41 39 41 41 44 46 44 45 46 50 49 50 50 53 55 55 56 58 58 57 61 61 62 64 62 65 65 65 69 69 71 72 73 71 72 73 74 78 79 77 81 82 83 83 82 84 86 87 89 88 90 91 92 94 94 95 94 98
70 76 19 43 67 99 51 26 54 98 95 48 35 85 5 93 40 66 32 63 8 97 80 59 15 47 90 94 92 22 18 27 91 52 31 42 96 60 58 23 20 75 89 49 88 68 45 37 86 56 84 74 79 38 87 16 78 83 69 12 72 71 11 24 77 28 65 62 14 82 44 53 46 9 30 55 64 50 6 81 61 57 73 21 34 17 7 10 39 36 29 13 25 33 4 41 3 2 1
1000 1
0 1 0 2 4 5 6 4 4 6 10 9 11 12 13 11 12 16 14 15 20 16 18 23 20 21 26 23 28 25 27 30 27 31 34 34 34 37 38 35 36 39 40 38 43 40 43 45 46 48 46 46 49 50 51 50 55 55 58 57 55 60 58 63 63 60 62 63 65 68 68 66 68 70 72 75 74 72 73 76 80 80 81 78 84 83 86 84 88 87 89 90 92 90 94 90 95 97 96 96 98 101 101 102 103 105 105 104 107 106 106 107 107 110 110 111 115 117 117 117 117 121 122 119 121 123 126 124 124 127 129 130 131 128 133 133 134 134 138 135 135 137 139 142 139 143 141 146 148 148 149 147 151 151 149 150 156 155 154 155 157 161 159 160 161 163 161 166 168 164 169 171 167 173 174 170 174 173 178 178 176 178 180 182 183 184 184 183 183 184 186 188 191 191 194 192 196 192 198 196 196 201 198 203 200 200 205 203 207 204 210 207 211 208 209 211 211 214 213 219 218 217 222 219 222 223 225 224 225 224 227 227 227 233 230 233 235 235 237 239 240 240 238 243 240 242 245 245 245 248 246 249 252 253 249 253 253 254 257 255 256 257 259 259 260 261 261 265 264 265 268 267 271 272 274 275 274 275 275 277 280 279 278 282 284 283 281 286 286 284 287 287 292 292 294 294 293 296 294 299 297 296 297 299 303 301 302 304 304 308 308 308 307 313 314 310 315 312 314 319 315 318 319 319 320 325 324 324 323 328 325 327 329 331 331 331 331 332 334 339 335 336 337 338 342 340 341 344 347 349 345 348 352 352 352 350 352 355 356 356 356 361 357 359 361 364 362 365 367 367 366 366 370 370 373 372 371 374 374 375 378 376 379 382 384 384 384 385 388 388 386 389 392 392 390 395 393 395 398 394 400 399 398 398 402 402 403 406 406 406 409 406 407 413 412 412 415 417 418 419 417 421 420 422 421 425 426 426 426 428 429 429 427 428 429 432 433 433 433 438 436 438 441 443 442 444 444 446 446 445 446 450 449 451 454 452 451 453 457 458 455 461 458 461 461 463 466 467 467 467 466 467 471 470 471 473 476 477 477 477 476 478 479 479 484 481 486 487 483 484 487 486 488 491 490 492 496 494 494 498 500 501 502 500 503 503 501 505 504 504 507 508 509 508 510 514 515 514 516 517 518 518 519 522 523 523 522 524 523 528 525 527 531 531 532 532 532 534 535 535 535 539 537 540 543 540 544 546 548 546 545 550 551 548 549 552 551 556 556 556 559 556 562 562 562 562 562 566 566 564 569 566 570 572 571 574 576 575 573 576 578 577 582 582 583 584 583 587 586 588 585 587 591 590 594 591 591 597 597 596 598 601 597 598 599 604 606 603 607 606 608 606 612 611 614 615 612 615 613 619 617 617 622 622 622 620 624 626 626 624 629 630 632 629 634 634 635 632 634 639 635 641 638 638 643 641 643 644 645 645 647 651 650 648 649 652 651 653 658 658 656 660 657 662 663 661 661 664 668 669 665 669 670 673 670 675 672 675 678 678 677 678 678 682 680 682 683 687 684 684 686 688 688 690 692 690 693 697 694 696 697 698 699 703 704 704 706 704 706 704 709 711 709 711 713 712 713 715 716 716 719 719 722 723 721 720 722 726 723 726 727 729 732 730 729 733 731 736 734 739 735 737 741 743 741 740 743 743 746 747 748 748 751 751 752 752 756 752 753 759 756 759 757 758 761 760 761 762 763 768 767 767 767 773 773 771 774 777 777 775 777 776 778 783 783 783 786 785 788 786 786 788 789 788 789 795 793 797 797 796 796 796 802 801 800 802 806 802 806 809 810 809 812 808 810 810 813 814 813 816 817 820 820 822 820 820 821 824 825 826 829 827 828 830 834 835 835 833 837 839 839 839 838 843 839 844 846 844 844 845 847 851 850 850 850 852 853 852 856 855 859 859 859 860 860 864 864 866 867 869 866 867 869 868 872 873 873 875 875 878 879 876 882 881 879 883 886 882 884 884 890 891 887 893 892 892 892 896 894 895 900 900 900 902 899 901 902 904 906 908 908 909 909 908 912 912 913 912 915 917 915 918 919 921 920 923 922 924 926 927 929 931 927 931 930 934 933 933 938 937 940 939 939 943 941 943 942 946 945 945 946 951 951 953 949 955 951 957 958 955 956 957 962 961 959 963 964 964 964 967 965 971 970 971 970 973 974 977 978 977 975 981 981 978 984 984 983 987 988 989 988 989 989 990 992 990 996 994 996
226 448 42 258 41 266 485 717 77 52 674 291 567 456 836 228 969 333 190 952 857 742 423 29 561 405 53 206 799 744 997 165 707 202 49 700 792 397 236 172 185 853 960 581 805 840 815 862 289 93 47 416 223 954 903 383 968 74 221 358 250 589 244 434 513 298 509 316 355 749 61 691 329 526 54 555 285 323 618 22 465 92 420 728 880 538 248 263 631 322 849 396 636 254 295 108 234 782 377 462 270 637 158 998 646 746 655 533 512 950 897 831 818 823 953 195 7 459 649 714 497 414 464 404 659 511 640 317 979 411 999 772 745 346 781 171 469 17 507 410 247 220 113 104 232 689 565 586 870 718 654 592 627 457 797 798 70 878 695 8 32 381 215 765 354 605 125 803 330 530 991 666 944 568 628 557 579 218 153 521 212 326 520 542 393 679 493 854 610 769 764 85 496 145 71 431 980 842 986 861 541 889 273 300 686 800 885 175 888 755 290 793 99 993 495 305 560 874 715 972 100 877 547 976 109 306 907 3 701 593 96 848 932 303 199 559 271 328 750 132 284 118 549 578 430 440 754 251 973 461 288 130 966 492 343 391 116 455 776 770 685 558 337 490 353 480 169 600 790 599 276 725 544 898 114 553 680 528 102 120 380 692 625 758 724 710 127 768 67 609 28 197 474 708 243 607 832 468 524 499 241 140 928 580 310 488 667 804 111 740 126 286 539 935 283 639 69 340 721 363 318 187 268 787 401 905 621 924 807 534 696 762 321 59 112 369 312 531 56 914 311 863 33 827 472 371 925 642 506 65 529 233 947 309 189 201 661 278 214 269 209 301 181 702 698 357 633 739 437 264 930 79 859 460 368 936 238 308 737 656 616 705 713 62 899 168 841 563 595 681 794 948 763 652 676 920 409 379 44 365 50 229 934 917 517 856 819 573 623 995 994 375 554 515 731 259 470 766 780 48 871 435 408 510 775 24 439 387 166 144 525 304 64 814 152 19 575 747 194 584 413 82 73 982 45 216 697 677 231 282 147 665 671 386 176 428 801 672 279 523 571 262 985 868 162 615 771 602 275 339 551 779 727 942 23 738 504 545 829 246 18 360 894 866 916 467 452 299 242 43 963 257 227 16 14 570 734 663 163 893 179 865 177 858 68 921 896 996 107 352 142 596 852 424 348 475 351 601 373 990 887 159 207 482 193 911 123 808 400 141 536 851 345 225 265 931 673 122 39 913 992 730 91 191 760 297 422 484 205 293 614 543 208 36 929 657 12 378 174 186 962 450 63 757 394 478 30 651 537 121 374 489 519 260 611 154 886 370 334 883 791 548 344 338 419 200 564 648 184 9 367 418 151 170 644 342 89 847 784 825 454 407 693 984 445 882 362 196 182 392 58 336 136 811 389 846 641 821 926 332 403 594 88 222 516 876 786 688 726 442 873 817 647 635 910 844 38 217 164 37 869 977 978 84 923 135 989 438 483 160 687 361 155 315 736 133 733 720 759 908 634 25 919 974 813 255 608 569 629 904 149 915 479 603 902 477 988 683 753 131 812 451 78 987 137 751 626 327 732 590 129 971 588 447 620 427 433 965 843 838 324 619 613 983 833 828 587 959 824 822 975 981 958 970 957 967 961 964 946 951 956 943 938 939 955 945 941 949 940 933 927 937 912 922 909 918 906 901 892 895 900 891 890 884 875 879 881 872 864 867 860 855 850 839 845 834 837 835 830 826 820 816 810 809 806 802 795 796 788 785 789 783 777 778 774 767 773 756 761 752 743 748 741 735 723 722 729 719 716 712 711 703 706 704 709 690 684 694 699 678 670 675 682 669 668 664 662 660 658 653 645 643 650 632 638 624 630 617 622 612 606 604 591 597 598 585 582 583 577 576 572 574 566 552 556 546 550 562 535 540 532 527 514 518 522 503 508 505 502 501 498 500 494 491 487 486 481 476 473 458 463 466 471 449 453 446 443 444 441 426 436 429 432 417 425 421 415 412 406 402 399 398 385 395 390 388 376 382 384 372 366 364 356 359 349 350 335 341 347 331 320 319 325 314 313 302 296 307 287 294 281 292 267 272 280 274 277 261 253 256 249 252 245 237 239 235 240 219 224 230 204 213 210 211 203 198 188 192 180 173 178 183 167 156 161 157 148 146 150 143 139 124 134 128 138 117 119 115 105 110 106 101 103 97 98 95 94 87 90 83 86 76 80 81 75 60 57 66 72 46 51 40 55 35 27 34 31 21 20 26 10 13 15 11 5 4 6 1 2
100 1
0 1 2 1 2 3 4 4 8 9 10 11 11 12 12 12 16 15 16 19 18 21 22 21 23 25 25 27 27 27 28 28 31 31 31 33 35 36 38 37 37 40 39 42 43 42 44 45 45 47 49 51 51 52 52 52 53 55 58 59 57 59 60 60 62 64 65 65 68 69 68 68 70 73 71 75 75 76 76 78 80 78 81 82 84 85 84 87 85 88 87 89 89 90 94 92 96 95 96
56 74 50 30 6 7 72 83 13 99 67 20 32 14 46 63 77 97 86 73 34 91 48 24 19 79 3 98 66 29 54 96 41 93 26 47 95 5 94 64 2 81 61 60 92 80 70 90 69 17 85 89 44 42 16 40 88 37 57 35 53 87 84 76 78 82 71 75 68 65 62 58 59 52 55 49 51 45 43 39 38 36 33 31 27 28 25 23 22 18 21 15 12 11 10 9 8 4 1
40 1
0 0 2 2 2 3 4 7 7 8 8 10 11 13 14 15 14 17 16 19 18 21 20 22 23 25 25 26 27 27 30 30 32 33 33 33 34 36 36
39 38 12 24 10 5 36 28 37 31 22 9 35 29 1 21 18 26 17 34 33 32 30 27 6 25 3 23 20 19 16 15 13 14 11 8 7 4 2
40 1
0 0 2 1 2 5 4 6 8 8 10 11 12 13 14 13 14 17 18 19 19 20 20 22 22 24 26 27 28 28 28 31 32 31 33 34 34 35 37
15 29 38 25 9 30 39 37 35 21 3 16 23 7 33 34 32 36 31 4 1 28 26 27 24 22 18 19 20 13 14 17 12 11 8 10 6 5 2
40 1
0 0 2 3 2 3 4 6 8 7 8 9 12 11 13 15 14 16 17 18 19 20 20 23 23 24 25 26 26 27 29 29 31 33 33 34 35 37 38
32 1 39 30 28 21 5 19 17 37 38 10 14 36 11 22 35 7 34 33 27 25 4 31 29 26 24 23 20 18 16 15 13 12 9 8 6 2 3
40 1
0 0 2 2 3 4 6 6 6 7 10 10 12 11 14 13 16 16 18 19 19 21 21 23 23 23 25 27 28 29 29 29 30 33 34 34 36 35 36
1 20 37 32 31 26 38 22 8 9 24 17 5 35 39 3 36 15 33 34 14 11 30 29 28 27 25 23 19 21 18 16 13 12 10 7 6 4 2
40 1
0 0 2 1 3 3 4 6 7 8 10 9 12 12 12 14 16 17 18 19 20 21 21 21 24 25 26 25 26 27 28 31 32 33 34 34 36 37 37
23 11 30 13 35 39 5 15 22 29 10 26 27 6 8 3 38 37 2 36 34 33 32 28 31 25 20 24 21 19 16 18 17 14 12 9 7 1 4
10 1
0 0 2 2 4 4 6 6 7
8 3 5 9 1 7 6 4 2
10 1
0 1 1 3 3 5 5 7 7
6 8 4 9 2 7 1 5 3
10 1
0 0 2 2 3 4 5 6 7
5 7 9 1 3 8 4 2 6
10 1
0 1 1 3 3 5 6 7 8
2 4 7 8 9 6 3 5 1
10 1
0 0 2 2 3 5 6 6 8
9 1 7 4 8 5 6 3 2
10 1
0 0 2 3 3 5 6 6 8
4 1 6 9 8 7 2 3 5
10 1
0 1 1 3 4 5 5 7 7
9 2 8 7 6 5 3 4 1
10 1
0 0 2 2 4 5 6 7 8
1 3 8 7 5 4 6 9 2
10 1
0 0 2 3 3 5 5 7 7
4 9 6 1 8 5 7 3 2
10 1
0 1 2 2 3 4 6 7 8
5 9 3 8 7 6 4 2 1
10 1
0 1 1 2 3 4 6 6 7
5 8 7 9 6 3 4 2 1
10 1
0 0 2 2 4 5 6 7 7
3 8 9 1 5 7 4 2 6
10 1
0 1 1 3 3 5 5 6 8
9 7 8 4 6 5 2 3 1
10 1
0 1 1 2 3 4 5 6 7
6 8 1 2 3 9 7 4 5
10 1
0 1 2 2 4 4 5 6 8
9 3 8 7 6 4 2 5 1
10 1
0 1 1 3 3 5 5 6 8
4 9 2 7 6 5 3 8 1
10 1
0 0 2 3 3 4 6 6 8
7 1 8 5 9 4 2 6 3
10 1
0 0 1 2 4 5 5 7 7
9 7 8 6 5 4 3 2 1
10 1
0 1 2 2 4 4 6 7 8
5 3 9 8 6 4 7 2 1
10 1
0 1 2 2 3 5 5 6 8
9 4 7 6 8 5 3 1 2
