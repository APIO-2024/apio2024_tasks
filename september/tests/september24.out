7ckgnn4wyi495puj3ibqf81dqvapyv6b
8
6
1
1
2
8
6
8
5
6
7
6
9
4
7
9
1
8
8
7
36
37
36
34
32
33
863
90
38
31
37
37
31
9
7
5
6
8
4
8
4
8
9
8
5
9
2
7
6
4
8
7
7
